protos/common.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/nivista/steady/.gen/protos/common";
+
+message Schedule {
+    oneof schedule {
+        CronConfig cron_config = 1;
+        IntervalConfig interval_config = 2;
+    }
+}
+
+message Task {
+    oneof task {
+        HTTPConfig http_config = 1;
+    }
+}
+
+message Meta {
+    google.protobuf.Timestamp create_time = 1;
+}
+
+message Progress {
```

Comment:
Is this progress struct specific to interval or will cron use the same struct?

Reply:
this is for everyone.

Reply:
Okay great